python_tests/test_python_frontend.py

```diff
@@ -2234,23 +2236,24 @@ def schedule(self):
     )
     def test_selected_device(self):
         """
-        Run the same Fusion as in test_scalar_only_inputs, but on device 1
+        Run the Fusion on device 1
         """
+        inputs = [
```

Comment:
bear with my ignorance, why do we need to change it to tensors here instead of just using scalar inputs?

Reply:
Previously, this test was a copy of `test-scalar-inputs`. The `exec-nvfuser` function checks if a new fusion is generated. There isn't a way to check if `test-scalar-inputs` is run prior to this test, so the new-fusion-generated check may fail. An easy fix was to slightly modify the fusion in this test so it is unique.

Reply:
got'ya. when seeing that we switched from scalar to tensor, I was under the impression that the device index here might not be tested as expected... I'm paranoid like I usually is :)